wiki/Help_centre/Silence/en.md

```diff
@@ -30,15 +30,25 @@ After a user is silenced, an alert will be displayed in place of the chat input
 
 If you believe you were silenced unfairly, contact the [account support team](/wiki/People/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) to contest the silence. Make sure to do it from the email linked to your osu! account; mention your osu! username, as well as the approximate date and time when the silence was handed out.
 
-## Common reasons for silences
+## Common reasons for silences and durations
 
 *Caution: Engaging in "inappropriate behaviour" in private messages is not against the rules, unless said inappropriate behaviour is being delivered to unwilling or unconsenting recipients. If reported, appropriate punishments will be placed.*
 
-- Inappropriate behaviour in public chats or forums (e.g. `#multiplayer`, beatmap discussions, comments, etc.), including — but not limited to — spamming, nonsense chat, racism, etc.
-- Uploading or posting inappropriate content via other means (e.g. profile avatars, beatmap submissions, profile details, user page bio, banner, etc.).
+| Silence reason | Duration |
+| :-- | :-- |
+| Spam or inappropriate discussions in chat | 5 minutes |
+| Inappropriate comments, beatmap discussion posts, or forum posts | 80 minutes |
+| Racism in chat | 80 minutes |
+| Inappropriate links or visual content | 43 hours[^exact-duration] |
+| Severe misconduct | 43 hours[^exact-duration] |
 
-The length of a silence may vary based on the severity of the infringement. Silences can stack on top of each other, adding more time for each one. Please note that if an offence is heavy enough, or if a player is a repeat offender, it may result in an [account restriction](/wiki/Help_centre/Account_restrictions) instead of a silence.
+While these are the most common reasons, silences can last any amount of time between 5 minutes and 28 days if the moderators deem it necessary due to the severity of the offence.
 
-## References
+Unless the new offence is more serious, a new silence will generally be twice the length of the previous silence, if any. Silences can stack on top of each other, adding more time for each one.
+
+Please note that if an offence is heavy enough, or if a player is a repeat offender, it may result in an [account restriction](/wiki/Help_centre/Account_restrictions) instead of a silence.
```

Comment:
```suggestion
Very severe or repeated offences may result in an [account restriction](/wiki/Help_centre/Account_restrictions) instead of a silence.
```